wbooks/src/app/components/AppNavigator/index.tsx

```diff
@@ -2,15 +2,16 @@ import React from 'react';
 import { createStackNavigator } from '@react-navigation/stack';
 import BookList from '@screens/BookList';
 import BookDetail from '@screens/BookDetail';
-import { ROUTES } from '@constants/routes';
+
+import { SCREEN_OPTS } from './constants';
 
 const Stack = createStackNavigator();
 
 function AppNavigator() {
   return (
-    <Stack.Navigator>
-      <Stack.Screen name={ROUTES.BookList} component={BookList} options={{ title: 'Listado de libros' }} />
-      <Stack.Screen name={ROUTES.BookDetail} component={BookDetail} options={{ title: 'Detalle' }} />
+    <Stack.Navigator screenOptions={SCREEN_OPTS as object}>
+      <Stack.Screen name="BookList" component={BookList} options={{ title: 'Listado de libros' }} />
+      <Stack.Screen name="BookDetail" component={BookDetail} options={{ title: 'Detalle' }} />
```

Comment:
Use  Routes constant. Why you change this?

Reply:
i... don't know what happened. This was already changed

Reply:
Mmm 🤔 did you merge with master?